notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
<p>Please, load the "lab_black" extension here ("%load_ext lab_black) to make sure each cell comply with black formatting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Added